spec/services/articles/feed_spec.rb

```diff
@@ -153,8 +153,8 @@
   describe "all non-default experiments" do
     it "returns articles for all experiments" do
       new_story = create(:article, published_at: 10.minutes.ago, score: 10)
-      NON_DEFAULT_EXPERIMENTS.each do |_method|
-        stories = feed.default_home_feed_with_more_randomness_experiment
+      NON_DEFAULT_EXPERIMENTS.each do |method|
+        stories = feed.send(method)
```

Comment:
`.public_send` ?

Reply:
I knew I was using the wrong one. Thanks!
```suggestion
        stories = feed.public_send(method)
```